tools/dataMigration/migrateMetadata/getCol.ts

```diff
@@ -81,6 +81,7 @@ export const getCol = (props: {
       calculateFn: _getCalculateFn(colSpec, cycles, assessment),
       validateFns: _getValidateFns(colSpec, cycles, assessment),
       style,
+      classNames: {},
```

Comment:
Added this line because the metadata migration was failing: 

```
TSError: ⨯ Unable to compile TypeScript:
tools/dataMigration/migrateMetadata/getCol.ts:75:5 - error TS2322: Type '{ cycles: string[]; colType: ColType; index: string | number; colName: string; variableNo: Record<string, string>; calculateFn: Record<string, string>; validateFns: Record<...>; style: Record<...>; }' is not assignable to type '{ cycles: string[]; } & ColProps'.
  Property 'classNames' is missing in type '{ cycles: string[]; colType: ColType; index: string | number; colName: string; variableNo: Record<string, string>; calculateFn: Record<string, string>; validateFns: Record<...>; style: Record<...>; }' but required in type 'ColProps'.

75     props: {
       ~~~~~

  src/meta/assessment/col.ts:49:3
    49   classNames: Record<string, Array<string>> // classes by cycle uuid
         ~~~~~~~~~~
    'classNames' is declared here.
  src/meta/assessment/cycle.ts:2:3
    2   props: {
        ~~~~~
    The expected type comes from property 'props' which is declared here on type 'Col & { forceColName?: boolean; }'

    at createTSError (/Users/diegoperez/UN/UNECE/repos/fra-platform/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/Users/diegoperez/UN/UNECE/repos/fra-platform/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/Users/diegoperez/UN/UNECE/repos/fra-platform/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/Users/diegoperez/UN/UNECE/repos/fra-platform/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/Users/diegoperez/UN/UNECE/repos/fra-platform/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/diegoperez/UN/UNECE/repos/fra-platform/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19) {
  diagnosticCodes: [ 2322 ]
}

```

Reply:
If this is correct, please, just resolve the conversation